### ISSUE TITLE
Configurable locale.

### ISSUE THREAD

**Issue by nstdio** [myoung34/docker-github-actions-runner#77]
It would be nice to have configurable or predefined (en_US.UTF-8?) locale. Currently the output for `locale`  command is

```sh
root@50a8b097f635:/actions-runner# locale
LANG=
LANGUAGE=
LC_CTYPE="POSIX"
LC_NUMERIC="POSIX"
LC_TIME="POSIX"
LC_COLLATE="POSIX"
LC_MONETARY="POSIX"
LC_MESSAGES="POSIX"
LC_PAPER="POSIX"
LC_NAME="POSIX"
LC_ADDRESS="POSIX"
LC_TELEPHONE="POSIX"
LC_MEASUREMENT="POSIX"
LC_IDENTIFICATION="POSIX"
LC_ALL=

```

**Comment by myoung34** [comment]
If you want to find a way to get this into the base docker file I'll happily merge it

**Comment by myoung34** [comment]
Resolved by #78
